cmd/podman/main.go

```diff
@@ -40,6 +41,11 @@ func main() {
 		return
 	}
 
+	cgroupv2, _ := cgroups.IsCgroup2UnifiedMode()
+	if !cgroupv2 {
+		logrus.Warnf("WARNING! You are using cgroups-v1 which is not supported with Podman v5 and will be removed in a future version. Consider switching to cgroups-v2.")
+	}
```

Comment:
This is running on the remote client where such warning make little to no sense and it also will run before any setup code so it is impossible to silence this warning (i.e. `--log-level error` will not work) which makes it super obnoxious. If there are users who are intentionally still on cgroupsv1 for whatever reason this will be pretty hard on them for no reason.

And looking at this the remote client on windows and macos would always print this line which is super bad. This definitely has to be moved to way later in the stack I guess somewhere in  the libpod runtime setup would be better suited.

Reply:
>  If there are users who are intentionally still on cgroupsv1 for whatever reason this will be pretty hard on them for no reason.

Did you mean this part for remote users or all users?

> 
> And looking at this the remote client on windows and macos would always print this line which is super bad. This definitely has to be moved to way later in the stack I guess somewhere in the libpod runtime setup would be better suited.

Ack, I'll update it, thanks.

Reply:
> Did you mean this part for remote users or all users?

All users, I mean if (for whatever reasons) a local user is still on cgroupv1 getting a warning on every single command is super obnoxious. While I agree users should see such warning by default I do think they need a way to silence it.

Reply:
would an envvar suffice or do we need a field in containers.conf or something?

Reply:
env var would be fine for me

Reply:
done. PTAL